src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,8 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            System.out.println(Lottery.getRandomBall());
+        }
```

Comment:
Don't use static methods in your solution. Instead of calling a static method on the Lottery class, you should create an instance of the Lottery class and call the method on that instance.